data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/SubMutateAction.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.logstash.mapping;
+
+import com.amazon.dataprepper.model.configuration.PluginModel;
+import org.opensearch.dataprepper.logstash.model.LogstashAttribute;
+
+public interface SubMutateAction {
+    static String getLogstashName() {
```

Comment:
I think you can remove this static method entirely.

Reply:
I use `getLogstashName` in MutateMapper